server/app.py

```diff
@@ -63,12 +63,56 @@ def create_user():
 
 @app.route('/user', methods=['PUT'])
 def modify_user():
-    return 'UNIMPLEMENTED'
+    if request.mimetype != 'application/json':
+        raise Exception('Content-Type is not "application/json".')
+    j = request.get_json()
+    Session, engine = dbconnect(db_options)
+    session = Session()
+    userID = j.get('id')
+    matchingUser = session.query(User).filter(User.id == userID).one_or_none()
+    if matchingUser:
+        matchingUser.first_name=j.get('first_name')
+        matchingUser.last_name=j.get('last_name')
+        matchingUser.username=j.get('username')
+        matchingUser.school_id=j.get('school_id')
+        matchingUser.email=j.get('email')
+        session.flush()
+        session.commit()
+        return jsonify(
+            {
+                'id': matchingUser.id
+            }
+        )
+    else:
+        response = jsonify( {'code': 404,
```

Comment:
no need to add code in hte json payload, otherwise this looks good! good job!
also if you want to know why 404, this wiki page is a good short lil synopsis on status codes:
https://en.wikipedia.org/wiki/List_of_HTTP_status_codes
will probably be refrencing other codes in the future